WalletWasabi/WabiSabi/Backend/Models/Alice.cs

```diff
@@ -11,9 +12,10 @@ public Alice(Coin coin, OwnershipProof ownershipProof)
 			// TODO init syntax?
 			Coin = coin;
 			OwnershipProof = ownershipProof;
+			Id = CalculateHash();
 		}
 
-		public Guid Id { get; } = Guid.NewGuid();
+		public uint256 Id { get; }
```

Comment:
Is this gonna be used to verify if the content is unchanged? If yes where?

Reply:
Given the AliceId is deterministic the client software can compute it independently and without requiring the server. So, we can create the AliceClient instances even before registering the coins. This is allows us also to verify the coordinator received and compute the AliceId in the same way. We can verify that once the `RegisterInput` returns the AliceId calculated by the coordinator.

Reply:
Alright.